stream-chat-swift-docs/docs/guides/working-with-members.md

```diff
@@ -1,3 +1,223 @@
 ---
 title: Working with Members
----
\ No newline at end of file
+---
+
+## ChatChannelMember object
+Chat channel members are represented by `ChatChannelMember` model.
+It's a subclass of `ChatUser` and apart properties typical for users, it also contains such info as member's role (owner, admin, moderator or member), invitation and ban details.
```

Comment:
Let's move this down and rename the header "ChannelMember vs User", this section should (and does) explain the difference and why we need it

Reply:
A nice suggestion, added an explanation on "ChannelMember vs User", but didn't get the "move down" part, what do you propose exactly?